src/content/reference/domain-allowlist/index.md

```diff
@@ -73,6 +73,8 @@ Below is a list of the external domains we require access to for our clusters to
     - Opsgenie's API is used to send alerts.
 - `gitlab.com`
     - Grafana may download plugins from repositories hosted by Gitlab.
+- `grafana.com`
+    - Grafana downloads a plugin directly from its API.
```

Comment:
I don't follow - which API?

Reply:
it's not, it's to pull the macropower plugin from their plugin registry